dev/io.openliberty.org.eclipse.microprofile.rest.client.2.0/bnd.bnd

```diff
@@ -0,0 +1,49 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+# IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+
+Bundle-SymbolicName: io.openliberty.org.eclipse.microprofile.rest.client.2.0; singleton:=true
+
+Export-Package: \
+  org.eclipse.microprofile.rest.client;version=1.4,\
+  org.eclipse.microprofile.rest.client.annotation;version=1.1,\
+  org.eclipse.microprofile.rest.client.ext;version=1.2,\
+  org.eclipse.microprofile.rest.client.inject;version=1.2.1,\
+  org.eclipse.microprofile.rest.client.spi;version=1.1.1
+
+Import-Package: \
+	javax.annotation;version='[1.3,2)',\
+	javax.enterprise.context;version='[2.0,2.1)',\
+	javax.enterprise.inject;version='[2.0,2.1)',\
+	javax.enterprise.util;version='[2.0,2.1)',\
+	javax.inject;version='[2.0,2.1)',\
+	javax.ws.rs.core;version='[2.1,3)',\
+	org.eclipse.microprofile.rest.client;version='[2.0,2.1)',\
+	org.eclipse.microprofile.rest.client.spi;version='[2.0,2.1)',\
+	*
+
+Include-Resource: \
+  @${repo;org.eclipse.microprofile.rest.client:microprofile-rest-client-api;2.0}
+# TODO: when replacing 2.0-RC2 with 2.0, make sure to add EXACT:
```

Comment:
leads me to believe we don't need EXACT on the others for now? not sure. Enjoy the spam.